deps/chakrashim/src/v8isolate.cc

```diff
@@ -245,24 +248,20 @@ bool Isolate::GetHeapSpaceStatistics(HeapSpaceStatistics* space_statistics,
 
 MicrotasksScope::MicrotasksScope(Isolate* isolate, MicrotasksScope::Type type) {
   // CHAKRA-TODO: Figure out what to do here
-  CHAKRA_ASSERT(false);
 }
 
 MicrotasksScope::~MicrotasksScope() {
   // CHAKRA-TODO: Figure out what to do here
-  CHAKRA_ASSERT(false);
```

Comment:
For MicrotasksScope I don't think there's anything we'll need to do, but I'm still investigating the DisallowJavascriptExecutionScope ones.